shower.js

```diff
@@ -716,11 +752,7 @@ window.shower = (function(window, document, undefined) {
 	* @returns {Boolean}
 	*/
 	shower.isSlideMode = function() {
-		if (isHistoryApiSupported) {
-			return /^full.*/.test(url.search.substr(1));
-		} else {
-			return document.body.classList.contains('full');
-		}
+		return shower.state.get('mode', MODES.LIST) === MODES.FULL;
```

Comment:
Can there be a situation when you call `state.get('mode')` without a default value and it returns `undefined`?